library/src/image.cxx

```diff
@@ -290,6 +312,25 @@ void image::save(const std::string& path, SaveFormat format) const
   }
 }
 
+//----------------------------------------------------------------------------
+std::vector<unsigned char> image::saveBuffer(SaveFormat format) const
+{
+  switch (format)
+  {
+    case SaveFormat::PNG:
+      return this->Internals->SaveBuffer<vtkPNGWriter>();
+    case SaveFormat::JPG:
+      return this->Internals->SaveBuffer<vtkJPEGWriter>();
+    case SaveFormat::TIF:
+      throw write_exception("Cannot save to buffer in TIF format");
+      break;
```

Comment:
`break` unreachable after `throw` (unless C++ is kinky that way?)

Reply:
You're absolutely right!